packages/shared/src/extension/app.ts

```diff
@@ -255,6 +255,8 @@ export async function boot(opts: BootOpts): Promise<App> {
   configUpdatePort.onMessage.addListener(function (message, sender) {
     appLog.debug('Received message on "ConfigUpdate" port %O', message);
     if (message.type === 'ReloadApp') {
+      if (message.payload)
+        localStorage.setItem('config', JSON.stringify(message.payload));
```

Comment:
Browser extensions have "their own" localStorage for saving such configuration and we implemented [here](https://github.com/tracking-exposed/trex/blob/master/packages/shared/src/extension/db.ts)